src/report.sh

```diff
@@ -300,19 +310,21 @@ function save_data_to()
   show_data > "$path"
 }
 
-function report_parse()
+function parse_report_options()
 {
-  local raw_options="$*"
-  local day
-  local week
-  local month
-  local year
   local output
-  local reference=0
+  local reference='DAY'
+  local reference_count=0
+  local options
+  local long_options='day,week,month,year,output,statistics'
+  local short_options='o'
```

Comment:
The parameter `output | o` requires a value; in this case, you need to add `:` to indicate that.

Additionally, we need to handle values around `day, week, month, year`, but those are optional. For this reason, you can use `::`. However, we will need to use `=`  to set a value.